src/main/java/nl/hsac/fitnesse/util/RandomUtil.java

```diff
@@ -36,4 +36,14 @@ public String randomString(String permitted, int length) {
         }
         return result.toString();
     }
+
+    /**
+     * Return a random value from a given enum
+     * @param clazz The enum class to get a value from
+     * @return a randomly selected value from the enum
+     */
+    public <T extends Enum<?>> T randomEnum(Class<T> clazz){
+        int x = random.nextInt(clazz.getEnumConstants().length);
```

Comment:
Aha it was in his pull request :-) https://github.com/fhoeben/hsac-fitnesse-plugin/pull/18/files#diff-f2af36c40499962467875ec0d926437bR47